assembl/utils.py

```diff
@@ -8,8 +8,20 @@
 from assembl.graphql.utils import get_root_thematic_for_phase
 
 
+def filter_with_date(data, start_date=None, end_date=None):
+    """
+    Returns an array of objects(could be ideas or posts) filtered by start and end date.
+    @param: start_date datetime
+    @param: end_date datetime
+    """
+    if start_date is not None and end_date is not None:
+        return [i for i in data if i.creation_date >= start_date and i.creation_date <= end_date]
+    else:
+        return data
```

Comment:
Not very efficient. You should do this filter on the query level, not afterwards.